js/config/scoring.js

```diff
@@ -95,12 +113,18 @@ YoastSEO.AnalyzerScoring = function( i18n ) {
                     score: 2,
                     text: i18n.dgettext('js-text-analysis', "You\'re linking to another page with the focus keyword you want this page to rank for, consider changing that if you truly want this page to rank.")
                 },
+
+                /* translators: %2$s expand the amount of outbound links */
```

Comment:
"%2$s expand the amount of outbound links" --> "%2$s expands to the number of outbound links"